### PR TITLE
BaSP-MR-187: Activities cards modals

### PR DESCRIPTION
**Pull request by flordimonaco**
Modals done for every activity :)

### CODE REVIEW

File: src/Components/Users/Member/modals/bodybuildingcard.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          src={`${process.env.PUBLIC_URL}/assets/images/Bodybuilding.svg`}
+          alt="body-building"
+        />
+        <h2>Body Building</h2>
+        <p>
+          Embark on a transformative journey of muscle development and strength training. Our
+          bodybuilding program is designed to sculpt and define your muscles, enhance your physique,
+          and elevate your overall strength. Train with expert guidance, push your limits, and
+          unlock your full potential.
+        </p>
```
Comment (YhanahiRosenthal) [inline src/Components/Users/Member/modals/bodybuildingcard.jsx:19]: you should make a single component for all activities, because it brings data from DB

**Review by YhanahiRosenthal — CHANGES_REQUESTED** [review]
i commented something on bodybiuldingcard

**Review by YhanahiRosenthal — APPROVED** [review]
Good job!

**Review by IvanJukonis — APPROVED** [review]

**Comment by flordimonaco** [comment]
[MODAL - Activity Card](https://trello.com/c/fxfWRjbB/187-modal-activity-card)

**Pull request merged**